docs/how-to/integrate_sdcore_with_external_gnb.md

```diff
@@ -29,7 +29,7 @@ Either create a new `.tf` file, or add the following content to you existing `ma
 ```console
 module "gnb01" {
   app_name   = "gnb01"
-  source     = "git::https://github.com/canonical/sdcore-gnb-integrator//terraform"
+  source     = "git::https://github.com/canonical/sdcore-gnb-integrator//terraform?ref=v1.5"
   model_name = "gnb-integration"
   channel    = "1.5/edge"
```

Comment:
```suggestion
  channel    = "1.5/stable"
```
Might be worth checking for other occurrences all over the docs. WDYT?

Reply:
Good catch, I will check and push another commit before merging.

Reply:
I found another one and fixed it, thanks again!